cmd/collector/main.go

```diff
@@ -31,8 +31,16 @@ import (
 func main() {
 	var configPaths = pflag.StringSlice("config", []string{"./config.yaml"}, "the collector config path")
 	_ = pflag.String("log-level", "", "not implemented") // TEMP(jsirianni): Required for OTEL k8s operator
+	var showVersion = pflag.BoolP("version", "v", false, "prints the version of the collector")
 	pflag.Parse()
 
+	if *showVersion {
+		fmt.Printf("observiq-otel-collector version %s\r\n", version.Version())
+		fmt.Printf("commit: %s\r\n", version.GitHash())
+		fmt.Printf("built at: %s\r\n", version.Date())
+		os.Exit(0)
```

Comment:
nit: You can just return here since return will automatically do a `0` exit code. No need to use the os package.